diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/files/IndentationRule.kt

```diff
@@ -132,6 +133,7 @@ class IndentationRule(private val configRules: List<RulesConfig>) : Rule("indent
             val numBlankLinesAfter = lastChild.text.count { it == '\n' }
             if (lastChild.elementType != WHITE_SPACE || numBlankLinesAfter != 1) {
                 val warnText = if (lastChild.elementType != WHITE_SPACE || numBlankLinesAfter == 0) "no newline" else "too many blank lines"
+                val fileName = filePath.substringAfterLast(File.separator)
```

Comment:
The variable `filePath` is used only here. Maybe it's better here without a variable too?

Reply:
Here it helps not to overcomplicate the string template, so I think it's fine to leave a variable